tests/unit/test_media_async.py

```
@@ -410,10 +418,12 @@ async def test_update_media_multiple_fields(
             assert media.caption == "New caption"
 
     @respx.mock
-    async def test_update_media_not_found(self, strapi_config: StrapiConfig) -> None:
+    async def test_update_media_not_found(
+        self, strapi_config: StrapiConfig, mock_media_response: dict
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_media_response`

(ARG002)